LuaSkin/LuaSkin/Skin.m

```diff
@@ -423,22 +428,25 @@ - (int)registerLibrary:(const luaL_Reg *)functions metaFunctions:(const luaL_Reg
     int tmpRefTable = luaL_ref(self.L, LUA_REGISTRYINDEX);
     NSAssert(tmpRefTable != LUA_REFNIL, @"Unexpected LUA_REFNIL registering library: %@", fname);
 
+    // FIXME: This doesn't seem to ever be used, and it conflicts with our new NSUUID driven refTable lookups. remove it?
```

Comment:
@asmagill something I did in this patchset is add a libraryName argument to the basic version of `registerLibrary` - am I reading the code right and this fname stuff is a hacky way to figure out the name, that I could drop as part of this?

Reply:
I think its used in two places -- throwing actually useful errors when two modules try to register the same NSObject helpers and adding a name to the actual registry table so that its easier to identify which module/file uses which table returned by `debug.getregistry`.

I suppose if you're rewriting the register methods so that passing in a string is *required* this code could be removed. This was just a way to do it without having to break any sort of compatibility with existing or third party modules.